grails-app/controllers/org/pih/warehouse/api/ReplenishmentApiController.groovy

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright (c) 2012 Partners In Health.  All rights reserved.
+ * The use and distribution terms for this software are covered by the
+ * Eclipse Public License 1.0 (http://opensource.org/licenses/eclipse-1.0.php)
+ * which can be found in the file epl-v10.html at the root of this distribution.
+ * By using this software in any fashion, you are agreeing to be bound by
+ * the terms of this license.
+ * You must not remove this notice, or any other, from this software.
+ **/
+package org.pih.warehouse.api
+
+import grails.converters.JSON
+import grails.validation.ValidationException
+import org.codehaus.groovy.grails.web.json.JSONObject
+import org.pih.warehouse.core.Location
+import org.pih.warehouse.core.User
+import org.pih.warehouse.inventory.InventoryLevelStatus
+import org.pih.warehouse.inventory.Requirement
+import org.pih.warehouse.order.Order
+import org.pih.warehouse.order.OrderType
+import org.pih.warehouse.order.OrderTypeCode
+
+class ReplenishmentApiController {
+
+    def identifierService
+    def replenishmentService
+
+    def list = {
+        // TODO: How to distinguish replenishments from stock transfer orders?
+        List<Order> replenishments = Order.findAllByOrderType(OrderType.get(OrderTypeCode.TRANSFER_ORDER.name()))
```

Comment:
For now let's just call it REPLENISHMENT_ORDER.

Reply:
I just realized that we don't have a separate PUTAWAY_ORDER so let's keep this as a TRANSFER_ORDER until there's a need to distinguish. I think the OrderType will have details about whether it's a replenishment, transfer, or putaway.